emukungu/IOT
# feat(data-visualization): plot graphs for weather data

#### What does this PR do?
- Add data visualization of weather data

#### Description of Task to be completed?
- Consume the ThingsSpeak API to retrieve weather data
- Plot line graphs of the data using d3.js

#### How should this be manually tested?
1. Check out to the branch `ft-data-visualization`
2. Create a virtual environment for the project, run `virtual env`
3. Install dependencies using `pip install -r requirements.txt`
4. On terminal, run command `python manage.py runserver`

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Postman Documentation
- N/A

#### Screenshots (If applicable)
<img width="1440" alt="Screen Shot 2019-08-08 at 3 13 37 AM" src="https://user-images.githubusercontent.com/9901086/62679565-8f8f4700-b9bd-11e9-9878-eeaca509980d.png">
